examples/deferred_with_accumulator.cpp

```diff
@@ -18,8 +18,13 @@
      USA
 */
 
+#include <string.h>
+#include <sys/types.h>        // TODO for ssize_t
```

Comment:
Not sure why we have a TODO here. Can we express the TODO better or just resolve it?

Reply:
My bad. (I was intending to look up the proper header for that as that one didn't look right, but forgot to get back to it).